kubernetes/secrets/synapse-config/homeserver.yaml.erb

```diff
@@ -0,0 +1,89 @@
+server_name: "matrix.ocf.berkeley.edu"
+pid_file: /data/homeserver.pid
+public_baseurl: https://matrix.ocf.berkeley.edu/
+
+federation_ip_range_blacklist:
+  - '127.0.0.0/8'
+  - '10.0.0.0/8'
+  - '172.16.0.0/12'
+  - '192.168.0.0/16'
+  - '100.64.0.0/10'
+  - '169.254.0.0/16'
+  - '::1/128'
+  - 'fe80::/64'
+  - 'fc00::/7'
+
+listeners:
+  - port: 8008
+    tls: false
+    type: http
+    x_forwarded: true
+    resources:
+      - names: [client, federation]
+        compress: false
+
+admin_contact: 'mailto:help@ocf.berkeley.edu'
+
+acme:
+    enabled: false
+    port: 80
+    bind_addresses: ['::', '0.0.0.0']
+    reprovision_threshold: 30
+    # FIXME: is this acme section necessary?
+    domain: matrix.example.com
+    account_key_file: /etc/matrix-shared/acme_account.key
+
+# TODO: Change to Postgres
+database:
+  name: "sqlite3"
+  args:
+    database: "/data/homeserver.db"
+
+log_config: "/data/matrix.ocf.berkeley.edu.log.config"
+
+media_store_path: "/data/media_store"
+enable_registration: false
+
+registration_shared_secret: "<%= registration_shared_secret %>"
+enable_metrics: true
+report_stats: false
+
+app_service_config_files:
+   - /etc/matrix-bridge/appservice-registration-irc.yaml
+
+macaroon_secret_key: "<%= macaroon_secret_key %>"
+
+form_secret: "<%= form_secret %>"
+
+signing_key_path: "/etc/matrix-shared/matrix.ocf.berkeley.edu.signing.key"
+old_signing_keys:
+    # TODO: populate this section with the old key
```

Comment:
wait what

Reply:
When we swap keys, we can use the old signing key here if needed, though if we are starting fresh this doesn't seem necessary.

Reply:
... I deleted this section because I don't think this will be necessary.